app/controllers/public/home_controller.rb

```diff
@@ -21,8 +21,8 @@ def create
     total = 0
 
     # if bank is empty report and test model for additional errors
-    flash[:error] = nil
-    flash[:error] = I18n.t(:no_bank_provided, scope: 'activerecord.errors.subscribe') if params[:bank].blank? && params[:method] == 'IDEAL' && @member.educations.none? { |education| Study.find(education.study_id).masters }
+    flash[:error] = I18n.t(:deprecated, scope: 'activerecord.attributes.payments.payment-type') unless params[:method] == 'IDEAL' # Cash/PIN
```

Comment:
We can also completely remove the `method` form parameter, as it only has one option anyhow.
I don't think we are adding any other payment methods in the near future.